reactioncommerce/reaction
# Order filtering shipping status

Resolves #2906 

### How to test
- Make a couple of orders 
- Mark different shipping statuses for these orders
- Observe that orders are filtered correctly by shipping status
- Test that the shipping status filter works in tandem with the other filters and the order search
